staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer.go

```diff
@@ -141,7 +143,7 @@ func getPublicIPDomainNameLabel(service *v1.Service) string {
 	if labelName, found := service.Annotations[ServiceAnnotationDNSLabelName]; found {
 		return labelName
 	}
-	return ""
+	return serviceAnnotationDNSLabelNotFound
```

Comment:
could we return `labelName, found` here? then we could check the bool value of found, which is easier than string.

Reply:
sure. just wanted to refactor less code. but to return found is reasonable.